Automattic/wp-calypso
# Add Tld Validation Schemas to Data Layer

This PR adds mechanisms to load JSON Schemas to validation TLD contact info to facilitate #21524. We're just dumping the schemas into the state in this PR, there will be followup changes to the forms themselves to unify around using them.

To test:
1) apply D9157-code to the backend (or maybe don't first to check the error notification)
2) Domains -> Add -> Add a domain with custom validation logic - 'fr', 'uk' or 'ca'
3) Using the redux devtools or `getState().domains.management.validationSchemas.uk` Verify that the schema is loaded into the redux store.
